packages/manager/modules/carbon-calculator/src/dashboard/carbon-footprint/template.html

```diff
@@ -1,8 +1,22 @@
 <div class="row-fluid">
-    <p
-        class="carbon-balance"
-        data-translate="carbon_calculator_dashboard_billing_explanation"
-    ></p>
+    <div class="carbon-balance mb-4">
+        <p
+            class="mb-2"
+            data-translate="carbon_calculator_dashboard_billing_explanation"
+        ></p>
+        <ul
+            data-ng-repeat="serviceName in $ctrl.availableServices track by $index"
+        >
+            <li
+                data-translate="carbon_calculator_dashboard_billing_explanation_service"
+                data-translate-values="{ serviceName: serviceName }"
+            ></li>
+        </ul>
+        <br />
```

Comment:
Why do you set `margin-bottom: 0` and add a `<br>` element ? Maybe you should just adapt your css definition